cpp/mcap/include/mcap/read_job_queue.hpp

```diff
@@ -0,0 +1,117 @@
+#pragma once
```

Comment:
`read_job_queue.hpp` and `unit-tests.cpp` follow different naming conventions

Reply:
i'm not sure why, but hyphens in a C++ source name feels really wrong. maybe because within the language, `-` is an operator, but `_` can be used in a variable name